packages/front-end/pages/settings/index.tsx

```diff
@@ -189,6 +200,50 @@ const GeneralSettingsPage = (): React.ReactElement => {
     setSaveMsg(true);
   };
 
+  const highlightColor =
+    value.confidenceLevel < 70
+      ? "#c73333"
+      : value.confidenceLevel < 80
+      ? "#e27202"
+      : value.confidenceLevel < 90
+      ? "#B39F01"
+      : "";
+
+  const pHighlightColor =
+    value.pValueThreshold > 0.3
+      ? "#c73333"
+      : value.pValueThreshold > 0.2
+      ? "#e27202"
+      : value.pValueThreshold > 0.1
+      ? "#B39F01"
+      : "";
```

Comment:
Are these dark mode-friendly? It might be a better idea to add these to the `theme.scss` file and add dark mode equivalents. Also having these in the theme might make sense in case we need to reuse these colours.

Reply:
They work fine with dark/light mode - but I agree this isn't the best place for it.